src/test/scala/nl.knaw.dans.easy.bag2deposit/ddm/RewriteSpec.scala

```diff
@@ -321,20 +322,45 @@ class RewriteSpec extends AnyFlatSpec with SchemaSupport with Matchers with DdmS
     // content of the <inCollection> element is validated in CollectionsSpec.collectionDatasetIdToInCollection
   }
 
-  it should "add inCollection for other than archaeology" in pendingUntilFixed {
-    val ddmIn = ddm(title = "blabla", audience = "Z99000", dcmi =
+  // TODO try incollection when DDM has no dcmiMetatdata
+  it should "add inCollection to an empty dcmiMetadata for other than archaeology" in {
+    val ddmIn = ddm(title = "blabla rabarbera", audience = "Z99000", dcmi =
+        <ddm:dcmiMetadata/>
+    )
+    val transformer = new DdmTransformer(
+      cfgDir,
+      Map("easy-dataset:123" -> <inCollection>mocked</inCollection>)
+    )
+
+    transformer.transform(ddmIn, "easy-dataset:123").map(normalized) shouldBe Success(normalized(
+      ddm(title = "blabla rabarbera", audience = "Z99000", dcmi =
         <ddm:dcmiMetadata>
+          <inCollection>mocked</inCollection>
+        </ddm:dcmiMetadata>
+      )))
+    // content of the <inCollection> element is validated in CollectionsSpec.collectionDatasetIdToInCollection
+  }
+  it should "add inCollection and filter titles" in {
+    val ddmIn = ddm(title = "blabla rabarbera", audience = "Z99000", dcmi =
+        <ddm:dcmiMetadata>
+          <dct:alternative>blabla</dct:alternative>
+          <dct:alternative>rabarbera</dct:alternative>
+          <dct:alternative>asterix</dct:alternative>
+          <dct:alternative>asterix</dct:alternative>
+          <dct:alternative>asterix en obelix</dct:alternative>
+          <dct:alternative>blabla rabarbera ratjetoe</dct:alternative>
```

Comment:
The profile title makes part of this last alternative title. It is not removed, what is desired behavior?

Reply:
As discussed, we leave the profile-title as-is. So this is the desired behavior